_inc/client/setup-wizard/feature-toggle-group/content.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { translate as __ } from 'i18n-calypso';
+
+const featureGroups = {
+	security: {
+		title: __( 'Security' ),
+		details: __(
+			'Keep your site totally backed up, prevent unwanted intrusions, find issues with malware scanning, and stop spammers in their tracks.'
+		),
+		features: [ 'backups', 'scan', 'anti-spam', 'brute-force-protect', 'monitor', 'sso' ],
+	},
+	performance: {
+		title: __( 'Performance' ),
+		details: __(
+			'Load pages faster! Shorter load times can lead to happier readers, more page views, and — if you’re running a store — improved sales.'
+		),
+		features: [
+			'site-accelerator',
+			'lazy-images',
+			'search',
+			'infinite-scroll',
```

Comment:
Should Infinite Scroll be on this list? It's only supported by a minority of themes, and given that it loads extra resources I don't know if that could be qualified as a performance tool?

Reply:
See the second half of my reply [here](https://github.com/Automattic/jetpack/pull/15637/#issuecomment-629447678); it's only going to be shown to users who indicate they are using blogs where it is an appropriate recommendation.

I agree that it probably shouldn't be under performance, it's probably better under `Design & Publishing`. @jessefriedman what are your thoughts on that?